plugins/statistics.koplugin/main.lua

```diff
@@ -1123,6 +1126,15 @@ The max value ensures a page you stay on for a long time (because you fell aslee
                             UIManager:show(durations_widget)
                         end,
                         keep_menu_open = true,
+                    },
+                    {
+                        text = _("Freeze finished books statistics"),
```

Comment:
`Freeze statistics of finished books` sounds ever so slightly better to me, RFC?